src/flows/components/Sidebar.tsx

```diff
@@ -233,6 +244,23 @@ const Sidebar: FC = () => {
           title: 'Export to Client Library',
           menu: <ClientList />,
         },
+        {
+          title: 'Inject Secret',
+          disable: () => {
+            const {type} = flow.data.byID[id]
+
+            if (type === 'rawFluxEditor' && !!editorContext.editor) {
```

Comment:
instead of registering this item for all panel types and selectively hiding the option, use the register option for the sidebar in the rawfluxquery panel's view.tsx to attach the item when we use the flux panel. this allows for a cleaner separation between the panels, allowing us to depreciate entire folders of our code without tracing down where the code integrates with the rest of the system inn places like this. as a byproduct, it also cleans up the lazy loading bundles a bit. you can see that pattern in action here:
https://github.com/influxdata/ui/blob/master/src/flows/pipes/Visualization/view.tsx#L106-L127

Reply:
done